src/main/resources/data/minecolonies/colony/quests/tutorial/hospital.json

```diff
@@ -0,0 +1,157 @@
+{
+  "name": "Pill for the Ill",
+  "max-occurrences": 1,
+  "parents": [
+    "minecolonies:tutorial/university"
+  ],
+  "triggers": [
+    {
+      "type": "minecolonies:citizen",
+      "state": {
+        "notmatch": {
+          "finavquests": [
+            "minecolonies:tutorial/housing"
+          ]
+        }
+      }
+    },
+    {
+      "type": "minecolonies:citizen",
+      "state": { }
+    }
+  ],
+  "triggerOrder": "1 || 2",
+  "objectives": [
+    {
+      "type": "minecolonies:dialogue",
+      "target": 0,
+      "text": "Now that the university is finished, our first concern should be regarding our health. We should really get a hospital in this town.",
```

Comment:
Too many "should" I think
"Now that the university is finished, our next concern is our healthcare. We should really get a hospital in this town." (maybe?)